### ISSUE TITLE
Proxy set but not used when displying diagnostics page

### ISSUE THREAD

**Issue by cert-orangecyberdefense** [MISP/MISP#2755]
# This template is meant for bug reports, if you have a feature request, please be as descriptive as possible and delete the template

*If you would like to report a bug, please fill the template bellow*

### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | Debian 8.10
| OS version (client)       | 
| PHP version               | 7.1.
| MISP version / git hash   | 2.4.84
| Browser                   | Any

### Expected behavior

load the diagnostics page from server settings, after going to github.com grabbing some info (version? etc.)

### Actual behavior

Can't load the diagnostics page from server settings.
In limbo. Any other config page from server settings loads fine.

### Logs, screenshots, configuration dump, ...

Here are some traces (tcpdumped)

14:24:11.490492 IP myhost.mydomain.44451 > lb-192-30-253-113-iad.github.com.https: Flags [S], seq 1021051012, win 29200, options [mss 1460,sackOK,TS val 250508 ecr 0,nop,wscale 7], length 0
14:24:15.498499 IP myhost.mydomain.44451 > lb-192-30-253-113-iad.github.com.https: Flags [S], seq 1021051012, win 29200, options [mss 1460,sackOK,TS val 251510 ecr 0,nop,wscale 7], length 0
14:24:23.506517 IP myhost.mydomain.44451 > lb-192-30-253-113-iad.github.com.https: Flags [S], seq 1021051012, win 29200, options [mss 1460,sackOK,TS val 253512 ecr 0,nop,wscale 7], length 0

proxy settings:

OS : 
- export http_proxy='http://A.B.C.D:EEEE'
- export https_proxy=$http_proxy

GIT :
git config --global http.proxy http://A.B.C.D:EEE
git config --global https.proxy http://A.B.C.D:EEE
config.php
  'Proxy' =>
  array (
    'host' => 'A.B.C.D',
    'port' => EEEE,
    'method' => '',
    'user' => '',
    'password' => '',
  ),

**Comment by iglocska** [comment]
Indeed, it will fetch the version string from github via the git command
line tool - which doesn't take the proxy settings. Will think about a clean
way to handle this.

**Comment by RichieB2B** [comment]
If you call git like this it should work:
```
git -c http.proxy=http://my.proxy.local:3128 pull
```

**Comment by iglocska** [comment]
not sure I want to pass anything user configurable to the command line though.

**Comment by RichieB2B** [comment]
Certainly not without sanitation. Alternatively we can document how to set the git config for the httpd/apache user.

**Comment by iglocska** [comment]
think I will just browse the remote and get the latest commit that way. Will look into it. (I mean just fetch it directly)
